eng/tools/ToolDescriptionEvaluator/consolidated-prompts.json

```diff
@@ -315,11 +317,11 @@
     "Get the details of knowledge base <agent-name> in the Azure AI Search service <service-name>",
     "Get the details of knowledge source <source-name> in the Azure AI Search service <service-name>",
     "Get the schema configuration for knowledge index <index-name>",
-    "List all agents in my AI Foundry project",
+    "List all agents in my Azure AI Foundry resource",
```

Comment:
It seems that this file was updated based on this open PR: https://github.com/microsoft/mcp/pull/1031. If so, I would suggest holding on merging this PR, until that one is merged.